sherpa/utils/tests/test_logging.py

```diff
@@ -25,6 +25,12 @@
 
 
 def test_logging_verbosity_contextmanager(caplog):
+    # Previous tests might have changed this to a non-default
+    # level already. So, we first make sure the root logger
+    # is set right.
+    sherpalogger = logging.getLogger('sherpa')
```

Comment:
This is a bit nicer with

```
    with caplog.at_level(logging.INFO, logger='sherpa'):
        logger = logging.getLogger('sherpa.some_module')
        logger.warning('1: should be seen')
        assert len(caplog.records) == 1

        ... rest of the test
```

Reply:
I can also push a change for this one too.

Reply:
Sure, I'm happy to take it!